src/logic/community/community.ts

```diff
@@ -25,12 +26,50 @@ import {
 import { EthAddress, PaginatedParameters } from '@dcl/schemas'
 
 export function createCommunityComponent(
-  components: Pick<AppComponents, 'communitiesDb' | 'catalystClient' | 'communityRoles' | 'logs'>
+  components: Pick<AppComponents, 'communitiesDb' | 'catalystClient' | 'communityRoles' | 'logs' | 'peersStats'>
 ): ICommunityComponent {
-  const { communitiesDb, catalystClient, communityRoles, logs } = components
+  const { communitiesDb, catalystClient, communityRoles, logs, peersStats } = components
 
   const logger = logs.getLogger('community-component')
 
+  const getCommunityMembers = async (id: string, options: GetCommunityMembersOptions, userAddress?: EthAddress) => {
```

Comment:
Can we name this function differently so it does not have the same name as the exported one, `getCommunityMembers`?